tests/test_visitors/test_ast/test_conditions/test_multiline_conditions.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    MultilineConditionsViolation,
+)
+from wemake_python_styleguide.visitors.ast.conditions import IfStatementVisitor
+
+incorrect_and_conditions1 = """if some and (
+    other == 1
+):
+    ...
+"""
+
+incorrect_or_conditions1 = """
+if some or (
+    other==1
+):
+        ...
+"""
+
+incorrect_and_conditions2 = """if some and some_function(
+    other,
+):
+    ...
+"""
+
+incorrect_or_conditions2 = """
+if some or some_func(
+    other,
+):
+        ...
+"""

```

Comment:
Please, also include:
1. Correct conditions with multiple `and` and `or`
2. Correct conditions with `()` like function calls, prioritisation, and tuples
3. Incorrect conditions for multiple lines: 2, 3 and 4

Reply:
Complete